sql/ts/src/skdb-cli.ts

```diff
@@ -213,21 +213,21 @@ if (values.dev) {
   }
 
   const schemeAndRest = host.split("://", 2);
-  const hostAndPort = schemeAndRest[schemeAndRest.length - 1].split(":", 2);
+  const hostAndPort = schemeAndRest[schemeAndRest.length - 1]!.split(":", 2);
 
   if (hostAndPort.length < 1) {
     console.log("Invalid host");
     process.exit(1);
   } else if (hostAndPort.length === 1) {
     dbCreds = await getCredsFromDevServer(
-      hostAndPort[0],
+      hostAndPort[0]!,
       3586,
       values.db as string,
     );
   } else {
     dbCreds = await getCredsFromDevServer(
-      hostAndPort[0],
-      parseInt(hostAndPort[1]),
+      hostAndPort[0]!,
+      parseInt(hostAndPort[1]!),
```

Comment:
Should we use comparison with `undefined` for such cases?
```ts
const [host, port] = schemeAndRest[schemeAndRest.length - 1]!.split(":", 2);
if (host === undefined) {
  // same error
} else {
  dbCreds = await getCredsFromDevServer(
    host,
    port ? parseInt(port) : 3586,
    ...
```

Reply:
Done in new version.